Dockerfile

```diff
@@ -1,7 +1,8 @@
-ARG GO_VERSION=1.21-bullseye
+ARG GO_VERSION=1.21-bookworm
```

Comment:
I'm not sure if that's compatible with the driver installation code below we have, but let's see the results of the CI

Reply:
I fixed the problem that was preventing CI from building the container